charts/pravega-operator/templates/operator.yaml

```diff
@@ -1,21 +1,38 @@
+{{- if .Values.pravegaOperator.create }}
```

Comment:
Does this chart make sense at all if it doesn't install the operator?

Reply:
Right! Forgot to delete that, it used to be the Pravega chart.